Source/Core/Core/FreeLookManager.h

```diff
@@ -4,23 +4,30 @@
 
 #pragma once
 
+#include <chrono>
+#include <optional>
+
 #include "Common/CommonTypes.h"
 #include "InputCommon/ControllerEmu/ControllerEmu.h"
+#include "InputCommon/ControllerEmu/Setting/NumericSetting.h"
 
 class InputConfig;
 
 namespace ControllerEmu
 {
 class ControlGroup;
 class Buttons;
+class IMUGyroscope;
 }  // namespace ControllerEmu
 
 enum class FreeLookGroup
 {
   Move,
   Speed,
   FieldOfView,
-  Other
+  Other,
+  Rotation,
+  RotationExact,
```

Comment:
It might be better to rename the name of "Rotation" to "RotationIncremental" for consistency.

Reply:
Thank you!   Agreed.  I missed this in my initial pass but plan on fixing this in a subsequent pass.